spec/requests/api/v1/articles_spec.rb

```diff
@@ -0,0 +1,33 @@
+RSpec.describe Api::V1::ArticlesController, type: :request do
```

Comment:
this should be described as `"GET /api/v1/articles/:id"`

Reply:
ok